core/staking_verifier_test.go

```diff
@@ -543,21 +545,21 @@ func TestCV32(t *testing.T) {
 func TestCV33(t *testing.T) {
 	statedb, _ := state.New(common.Hash{}, state.NewDatabase(ethdb.NewMemDatabase()))
 	msg := createValidator()
-	statedb.AddBalance(msg.ValidatorAddress, big.NewInt(5e18))
-	// MinSelfDelegation < 1 ONE
-	msg.MinSelfDelegation = big.NewInt(1e18 - 1)
+	statedb.AddBalance(msg.ValidatorAddress, tenK)
+	// MinSelfDelegation < 10,000 ONE
+	msg.MinSelfDelegation = big.NewInt(1e18)
 	if _, err := VerifyAndCreateValidatorFromMsg(
 		statedb, postStakingEpoch, big.NewInt(0), msg,
 	); err == nil {
-		t.Error("expected", "delegation-given 999999999999999999: min_self_delegation has to be greater than 1 ONE", "got", nil)
+		t.Error("expected", "delegation-given 1000000000000000000: min_self_delegation has to be greater than 10,000 ONE", "got", nil)
```

Comment:
we can allow 10k OR greater? so that it's easier to create a validator at 10k self delegation, instead of 10k+1

Reply:
@john-harmony we should update https://docs.harmony.one/home/validators/first-time-setup/creating-a-validator to have 10k+1? currently, 

```
--validator-addr [ONE ADDRESS] --amount 10 \
--min-self-delegation has to be at least 10,000 ONE.
```